var/spack/repos/builtin/packages/amrex/package.py

```diff
@@ -133,3 +137,34 @@ def cmake_args(self):
             args.append('-DCMAKE_Fortran_MODDIR_FLAG=-M')
 
         return args
+
+    #
+    # For versions > 20.11
+    #
+    @when('@20.11:')
```

Comment:
```suggestion
    @when('@20.12:')
```
Version ranges are inclusive. As is, these would overlap.

Reply:
Thanks for noticing this. What about `develop`? Will it use the "new" options set or the old one?

Reply:
`develop` sorts as greater than all numbered releases